hooks/pre-push

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 The TIE Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+#     Not meant for manual use
+#     Runs automatically before a git push command is executed
+#     Behaviour can be set up using "bash scripts/setup_hooks.sh"
+
+# Ask user if push should continue anyway if a test fails
+
+proceed_when_test_fail () {
+    # Assign standard input to keyboard
```

Comment:
I think we use 2-space indentation for all our scripts, let's just do that in general to keep things consistent.